index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <title>Home</title>
+</head>
+<body id="body">
+    <!-- first page for home -->
+    <header class="header" id="home">
+        <nav class="nav-top-desktop">
+            <ul>
+                <li><i class="fa-brands fa-facebook-f"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <nav>
+            <div class="nav-top-mobile">
+                <button id="wrapper"><i class="fa-sharp fa-solid fa-bars"></i></button>
+            </div>
+            <div class="nav-wrapper" id="nav-wrapper">
+                <a class="close-button" id="close-wrap"><i class="fa-solid fa-xmark"></i></a>
+                <div>
+                    <a href="">Home</a>
+                    <a href="about.html">About</a>
+                    <a href="">Program</a>
+                    <a href="">Join</a>
+                    <a href="">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="">IOT Campaign</a>
+                </div>
+            </div>
+            <div class="nav-wrapper-desktop">
+                    <h1>IOT-Horizons</h1>
+                <div>
+                    <a href="about.html">About</a>
+                    <a href="">Program</a>
+                    <a href="">Join</a>
+                    <a href="">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="">IOT Campaign</a>
+                </div>
+            </div>
+        </nav>
+        <div class="presentation-container">
+            <div>
+                <h3>"Hello Tech world"</h3>
+                <h1>IOT CONGRESS 2023</h1>
```

Comment:
- [ ] I think it would be nicer if you have your project own logo here since you are just putting it as headung but it is a logo in the Given page and I think that would be great addition to your page 🙏🏻💯🚀🤩🤩 You can make it with canvas it is also found in your dashboard GitHub Education so you can get it for free for a period 🤩🚀
### It would be great experince if you try it now because you are going to need it in the future 💯💯